src/tsx/coreMessages.tsx

```diff
@@ -0,0 +1,9 @@
+interface LiveAction {
```

Comment:
What does "Live" mean here? Are there other types of Action?

Reply:
I was thinking that the Actions in the Opus are preprogrammed but here they are assembled "Live" instead. I wanted a distinction because they are handled (slightly) different but still calling it Action something since they contain the same data as an Action in the opus. Looking back at it, I'm (very) not too happy with the name. What do say about calling it _Action_ here as well?

Reply:
I think Action is good if it looks the same as an action in the opus. Otherwise, maybe OnTheFlyAction to match the method name in Core?